### DISCUSSION TITLE
how can I export a Kimai reporting pie chart to excel?

### DISCUSSION THREAD

**Discussion by fipio** [kimai/kimai#3935] [Questions & Answers] [answered]
In reporting session of Kimai, where I see a pie chart of summary under a project like this:

Activity 1: 20 hours in total
Activity 2: 3 hours in total
...

Can I export these into excel spreadsheets?

**Comment by kevinpapst** [comment, marked as answer]
Unfortunately not yet.
There is no native Export function for the Project detail report.
All you can try is to mark/highlight the table/data with your mouse and try to copy & paste it into Excel.
